### ISSUE TITLE
How to keep connection open untill the upload is complete using DCM/ANT.

### ISSUE THREAD

**Issue by RahulRdec** [IBM-UrbanCode/Datapower-Configuration-Manager-UCD#4]
Hi All,

Te Datapower device is having default 4 min session timeout. To work on UI, I used to enable retry add-on to  keep my session refreshed after each 4 mins. but when I am logging in using ANT/DCM, it is getting logout after 4 mins without allowing command steps to complete. 

Is there any way I can keep my session active till the command execution is complete?

For example, I am uploading 23 files to the server but every time it is getting failed after 4 mins (only 5 files are getting upload) and saying Authentication error.

**Comment by richgroot** [comment]
That's a very slow upload!  Each file upload is a separate transaction, so each one needs to complete within its own 4 minute window.  Are you running out of space on DataPower?  I explained the Authentication Failure message in another issue, but it is a pretty general message that simply means that the request failed for some reason.  You'll need to do some diagnosis to track down the problem.

You might also try uploading the file in the DP web browser.  That sometimes gives a little more information about the failure.

**Comment by RahulRdec** [comment]
No, there is no issue of space and while trying to upload using browser, there is no issue as well. Is there any way we can control this connection timeout using ANT/DCM function?
